R/opts.R

```diff
@@ -442,6 +461,98 @@ opt_table_outline <- function(data,
   # Perform input object validation
   stop_if_not_gt(data = data)
 
-  values_vec <- rep(c(style, width, color), 4)
-  tab_options_multi(data, create_option_value_list(table_borders_vec(), values_vec))
+  if (style == "none") {
+    width <- NULL
+    color <- NULL
+  }
+
+  params <-
+    c(style = !is.null(style), width = !is.null(width), color = !is.null(color)) %>%
```

Comment:
```r
c(!is.null(style) "style", !is.null(width) "width", is.null(color) "color")
```

Reply:
I couldn't get the suggested line to work (`Error: unexpected string constant in "c(!is.null(style) "style""`). The below does work though:

```
  params <-
    c(
      if (!is.null(style)) "style" else NULL,
      if (!is.null(width)) "width" else NULL,
      if (!is.null(color)) "color" else NULL
    )
```

Reply:
Yep that's right. The `else NULL` is optional.